### PR TITLE
Tenet Garden ---> Gate Route - scrnshot edit

### PR DESCRIPTION
**Pull request by vtange**
Caught what I could amid the text glitches. Focused on obvious/ugly moments of bad translation (like during cutscenes)

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -2551,8 +2551,8 @@ ETC_20150317_002550	{memo X}Obtained Mayvern's key!
 ETC_20150317_002551	{memo X}Naktis' subordinates are being too disrupting!
 ETC_20150317_002552	{memo X}Using a key of Mayvern..
 ETC_20150317_002553	Experimental Mesyum
-ETC_20150317_002554	Acidic solution Maki
-ETC_20150317_002555	While using to change the horsepower
+ETC_20150317_002554 	$Applying Acidic Solution
```
Comment (Sunaries) [inline ETC.tsv]: Applying Acidic Solution -> Sprinkling the Acidic Solution

File: ETC.tsv [hunk 2]
```diff
@@ -2551,8 +2551,8 @@ ETC_20150317_002550	{memo X}Obtained Mayvern's key!
...
+ETC_20150317_002555	$Charging Magic Power
```
Comment (Sunaries) [inline ETC.tsv]: Charging Magic Power -> Using the Magic Crystal

File: ETC.tsv [hunk 3]
```diff
@@ -4765,7 +4765,7 @@ ETC_20150317_004764	{@st54}{b} Select Reward{/}
 ETC_20150317_004765	{memo X}Nudoku I was mounting the Son Bol pocket in Kochi.
 ETC_20150317_004766	Nudoku is available in Kochi!
 ETC_20150317_004767	Can be used in the tree Ferri!
-ETC_20150317_004768	Horsepower charge (
+ETC_20150317_004768	$Magic Power Charge (
```
Comment (Sunaries) [inline ETC.tsv]: Magic Power Charge ( -> Magic charged (

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**